packages/render-fidelity-tools/test/config.json

```diff
@@ -358,6 +358,166 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-ToyCar",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/ToyCar/glTF-Binary/ToyCar.glb",
+      "orbit": {
+        "radius": 0.037,
+        "theta": -35,
+        "phi": 75
+      },
+      "dimensions": {
+        "width": 1200,
+        "height": 500
+      },
+      "target": {
+        "y": 0.001
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-IridescentDishWithOlives",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/IridescentDishWithOlives/glTF-Binary/IridescentDishWithOlives.glb",
+      "orbit": {
+        "radius": 0.47
+      },
+      "dimensions": {
+        "height": 570
+      },
+      "target": {
+        "y": 0.11
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-TextureTransformMultiTest",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/TextureTransformMultiTest/glTF-Binary/TextureTransformMultiTest.glb",
+      "orbit": {
+        "radius": 2.9,
+        "phi": 98
+      },
+      "target": {
+        "y": -0.28
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-SheenChair",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/SheenChair/glTF-Binary/SheenChair.glb",
+      "orbit": {
+        "radius": 1.0
+      },
+      "target": {
+        "y": 0.3
+      },
+      "dimensions": {
+        "height": 450
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-SheenCloth",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/SheenCloth/glTF/SheenCloth.gltf",
+      "orbit": {
+        "radius": 0.059
+      },
+      "target": {
+        "y": 0.015
+      },
+      "dimensions": {
+        "width": 1200
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-MaterialsVariantsShoe",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MaterialsVariantsShoe/glTF-Binary/MaterialsVariantsShoe.glb",
+      "orbit": {
+        "radius": 0.275,
+        "theta": 35,
+        "phi": 49
+      },
+      "target": {
+        "y": 0.06,
+        "x": 0.03
+      },
+      "dimensions": {
+        "height": 500
+      },
+      "exclude": [
+        "dspbr-pt"
+      ]
+    },
+    {
+      "name": "khronos-MetalRoughSpheresNoTextures",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MetalRoughSpheresNoTextures/glTF/MetalRoughSpheresNoTextures.gltf",
+      "orbit": {
+        "radius": 120.0
```

Comment:
Any idea why this one wasn't working? I'm assuming it could use the same configuration from the other MetalRoughSpheres examples we have (and it could probably live next to them).

Reply:
I'm not really sure what is going on. Based on the picture, I thought maybe I'm too close, but even after playing around with Z and radius, I couldn't make it work. I did move it next to the other ones.

Reply:
I just loaded this in the editor and if you click "save current as initial" under the camera tab, you'll see the default radius is 0.015m in the snippet. This must be another accidentally tiny model.